desktop/main.js

```diff
@@ -183,7 +183,7 @@ const mainWindow = (() => {
                 const path = require('path');
                 const devEnvConfig = dotenv.config({path: path.resolve(__dirname, '../.env')}).parsed;
 
-                if (devEnvConfig.USE_WEB_PROXY === 'true') {
+                if (devEnvConfig.USE_WEB_PROXY !== 'false') {
```

Comment:
Hey, sorry you still have to have an `.env` file to make this work 
When you don't have an `.env` file `parsed` - `devEnvConfig` is `undefined and you get this error: 

```
[Main] 14:10:46.010 › (node:8157) UnhandledPromiseRejectionWarning: TypeError: Cannot read properties of undefined (reading 'USE_WEB_PROXY')
[Main]     at /Users/kidroca/Desktop/Freelance/Expensify.cash/desktop/main.js:186:34
```

What the [proxy script](https://github.com/Expensify/App/blob/fd24e0d1b61b90058f2a1888fb1fb103e470f4cb/web/proxy.js#L3-L5) does is a bit more flexible and you don't need to import `path`
```js
dotenv.config();

if (process.env.USE_WEB_PROXY !== 'false') ...
```

When you don't pass a path to dotenv it will use `.env` by default
If there is such a file the variables defined would be assigned to process.env

Reply:
Whoops ... I encountered this locally and had committed but not pushed the fix 😞